stock-tracker/src/features/header/component/priceDisplay.tsx

```diff
@@ -20,9 +21,11 @@ export const PriceDisplay = () => {
       >
         <Icon
           className="price_display__arrow_icon"
-          type={response.change < 0 ? "arrow-down" : "arrow-up"}
+          type={response && response.change < 0 ? "arrow-down" : "arrow-up"}
         />
-        {response.change ? Math.abs(response.change.toFixed(2)) : ""}{" "}
+        {response && response.change
+          ? Math.abs(response.change).toFixed(2)
+          : ""}{" "}
```

Comment:
nit: What's the `{" "}` for?

Reply:
Not sure, I think prettier might have added them at some point but will remove.